graphql/resolvers/addComment.test.js

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
NIT: space between the test cases and variables would make the code a little bit easier to go through.